common/mails/courses/index.ts

```diff
@@ -234,40 +234,64 @@ const isPromotionValid = (publishedAt: Date, capacity: number, alreadyPromoted:
     return capacity < 0.75 && alreadyPromoted === false && (daysDiff === null || daysDiff > 3);
 };
 
-export async function sendPupilCourseSuggestion(course: Course | Prisma.course, subcourse: Prisma.subcourse, actionId: ActionID) {
-    // TODO: Reenable once we stabilized course promotions
-    return;
+function shuffleArray<T>(array: T[]): T[] {
```

Comment:
Could be moved to /common/util

Reply:
Also just for fun: This could be implemented slightly more efficient ;) https://github.com/jashkenas/underscore/pull/2158 ... However for a few thousand elements it probably won't matter

Reply:
so you mean using a new approach right? because I dont see how you could tweake the current approach to make it more efficient. ([reference for current shuffle](https://stackoverflow.com/questions/2450954/how-to-randomize-shuffle-a-javascript-array))